src/apps/Agreement/DisputableApps/DisputableApps.js

```diff
@@ -0,0 +1,221 @@
+        padding-top: ${3 * GU}px;
+        padding-bottom: ${3 * GU}px;
+      `}
+    >
+      <div
+        css={`
+          display: inline-grid;
+          grid-template-columns: ${layoutName === 'small' ? '1fr' : '1fr 1fr'};
+          column-gap: ${8 * GU}px;
+        `}
+      >
+        <div>
+          <InfoField
+            label={
+              <React.Fragment>
+                Action Collateral
+                <Help hint="What is Action Collateral?">
+                  <strong>Action Collateral</strong>.
+                </Help>
+              </React.Fragment>
+            }
+            css={`
+              margin-bottom: ${EXPANDABLE_ROW_GAP};
+            `}
+          >
+            <TokenAmount
+              address={actionCollateral.address}
+              symbol={actionCollateral.symbol}
+              amount={actionCollateral.amount}
+            />
+          </InfoField>
+          <InfoField
+            label={
+              <React.Fragment>
+                Challenge Collateral
+                <Help hint="What is Challenge Collateral?">
+                  <strong>Challenge Collateral</strong>.
+                </Help>
+              </React.Fragment>
+            }
+            css={`
+              margin-bottom: ${EXPANDABLE_ROW_GAP};
+            `}
+          >
+            <TokenAmount
+              address={challengeCollateral.address}
+              symbol={challengeCollateral.symbol}
+              amount={challengeCollateral.amount}
+            />
+          </InfoField>
+        </div>
+        <div>
+          <InfoField
+            label={
+              <React.Fragment>
+                Challenge Period
+                <Help hint="What is Challenge Period?">
+                  <strong>Challenge Period</strong>.
+                </Help>
+              </React.Fragment>
+            }
+            css={`
+              margin-bottom: ${EXPANDABLE_ROW_GAP};
+            `}
+          >
+            {settlementPeriod} <SubtleLabel>Hours</SubtleLabel>
+          </InfoField>
+
+          <InfoField
+            label={
+              <React.Fragment>
+                Settlement Period
+                <Help hint="What is Settlement Period?">
+                  <strong>Settlement Period</strong>.
+                </Help>
+              </React.Fragment>
+            }
+            css={`
+              margin-bottom: ${EXPANDABLE_ROW_GAP};
+            `}
+          >
+            {challengePeriod} <SubtleLabel>Hours</SubtleLabel>
+          </InfoField>
+        </div>
+      </div>
+
+      <InfoField
+        label="Signer Eligibility"
+        css={`
+          margin-bottom: ${EXPANDABLE_ROW_GAP};
+        `}
+      >
+        Open to tokenholders with a minimun token balance of{' '}
+        {signerEligibility.amount}{' '}
+        <div
+          css={`
+            display: inline-block;
+            position: relative;
+            top: 3px;
+          `}
+        >
+          <TokenBadge
```

Comment:
We should check with @dizzypaty, but in this case, it's likely we want the inline version of the badge as it's inline with text.

But related to a comment above, we may also want to think about how to express token amounts in this badge.

Reply:
What is meant by the "inline" version? I can't find this referenced anywhere.

Reply:
The last one in [the examples here](https://devbox.dweb.cool/#TokenBadge):

<img width="265" alt="Screen Shot 2020-06-30 at 2 01 03 PM" src="https://user-images.githubusercontent.com/4166642/86123655-2d11ce80-bada-11ea-8295-4bc88de233ff.png">

Reply:
Noice, that's what I went with :)